workspaces/frontend/src/app/pages/Workspaces/Workspaces.tsx

```diff
@@ -129,6 +134,8 @@ export const Workspaces: React.FunctionComponent = () => {
     state: 'State',
     homeVol: 'Home Vol',
     dataVol: 'Data Vol',
+    cpu: 'cpu',
```

Comment:
For the column name, it should be uppercase
```suggestion
    cpu: 'CPU',
```

Reply:
Fixed